src/cpp/src/visual_language/input_embedder.cpp

```diff
@@ -41,6 +41,19 @@ class InputsEmbedder::IInputsEmbedder {
 public:
     virtual ov::Tensor get_inputs_embeds(const std::string& prompt, const std::vector<ov::Tensor>& images) = 0;
 
+    ov::Tensor get_inputs_embeds(ov::Tensor input_ids) {
+        m_embedding.set_input_tensor(input_ids);
+
+        m_embedding.infer();
+        const ov::Tensor& embed_prompt_tensor = m_embedding.get_output_tensor();
+        float* embed_data = embed_prompt_tensor.data<float>();
+        for (auto idx = 0; idx < embed_prompt_tensor.get_size(); idx++) {
+            embed_data[idx] = embed_data[idx] * m_vlm_config.scale_emb;
```

Comment:
can we merge it as post-processing step to the model itself?
Example https://github.com/openvinotoolkit/openvino.genai/blob/master/src/cpp/src/text2image/models/autoencoder_kl.cpp#L93-L115

I think you can do it via `custom` post-processing

Reply:
thank you ! I will do